### ISSUE TITLE
Ruuvitag Data Spikes

### ISSUE THREAD

**Issue by askpatrickw** [victronenergy/venus#1399]
I see these spikes in my Ruuvitag data.

![image](https://github.com/user-attachments/assets/d45ae6fe-af25-4ebe-8636-2e01bc574a65)

I found a 2022 related thread, but nothing in this repo about a fix.

https://communityarchive.victronenergy.com/questions/119966/spikes-in-data-collected-from-ruuvitags.html

**Comment by askpatrickw** [comment]
I updated the description to note that I'm pulling this data into Home Assistant over MQTT.

**Comment by askpatrickw** [comment]
A few of us over on the Ford Transit USA Forum have noticed it is not just Ruuvi, but any values collected over MQTT.

I see this in the SOC for example.

<img width="757" alt="image" src="https://github.com/user-attachments/assets/f9ebc34c-6152-432b-b8f7-5f61fa8c041e">

**Comment by askpatrickw** [comment]
I don't think which sensor (in HA terminology) matters.

**Comment by askpatrickw** [comment]
And I'm evening rounding the value in the case of SOC.

```json
  - name: "Battery SOC"
    unique_id: "HX4MNJHAXmW_bkFhrQPYt"
    device_class: battery
    state_topic: "victron/N/c0619ab1f84c/system/0/Dc/Battery/Soc"
    suggested_display_precision: 0
    value_template: "{{ value_json.value|round(0) }}"
    unit_of_measurement: "%"
    device:
      {
        identifiers: ["Cerbo GX"],
        manufacturer: "Victron Energy",
        model: "Cerbo GX",
        name: "Cerbo GX",
      }
```

**Comment by dirkjanfaber** [comment]
@mpvader : We could track the SOC and temperature value of that system via docker/grafana. If that shows spikes, we know the cause is somewhere in mqtt. Otherwise the problem must be on the HA side.

**Comment by dirkjanfaber** [comment]
@askpatrickw : I've set up the mqtt logging on our side. Please report future spikes with the (approximate) timestamp in this call. It might still take a  while before we can look into it, but at least we know where to look then.

**Comment by askpatrickw** [comment]
@dirkjanfaber Not sure I follow. How will this logging work exactly?

Maybe you mean you turned that on in VRM?
I don't use VRM...

**Comment by dirkjanfaber** [comment]
Indeed. The site is setup to log to VRM, even if you don't use that. Of course you can switch that off, but as it might help us find the cause for the spikes, I'd like you to leave that on while this issue is still open.

**Comment by askpatrickw** [comment]
@dirkjanfaber OK... actually it has been on.
Around 6pm Pacific there are bottom spikes.

**Comment by dirkjanfaber** [comment]
Is it possible for you to set the _VRM Online portal -> VRM Portal_ value to ***Full*** for a while? 

<img width="1325" alt="Image" src="https://github.com/user-attachments/assets/aad3efa8-fc37-454b-8a50-517a168758de" />

**Comment by askpatrickw** [comment]
@dirkjanfaber I've turned it on.

**Comment by dirkjanfaber** [comment]
@askpatrickw  I've been logging for the last 2 days, but don't notice any spikes in the mqtt data. Have you seen any spikes lately?
